php/class-delivery.php

```diff
@@ -820,6 +820,8 @@ protected function standardize_tag( $tag_element ) {
 			$tag_element['atts']['data-permalink'] = add_query_arg( 'asset', $tag_element['id'], $base_url );
 		}
 
+		$tag_element['atts']['data-version']  = $this->media->get_post_meta( $tag_element['id'], Sync::META_KEYS['version'], true );
```

Comment:
@pereirinha we have a method for that, which add the 1 if missing 😉 
```suggestion
		$tag_element['atts']['data-version'] = $this->media->get_cloudinary_version( $tag_element['id'] );
```